sdk/core/Azure.Core/src/DiagnosticsOptions.cs

```diff
@@ -26,10 +30,21 @@ internal DiagnosticsOptions()
             LoggedQueryParameters = new List<string>();
         }
 
+        /// <summary>
+        /// Get or sets value indicating whether HTTP pipeline logging is enabled.
+        /// </summary>
         public bool IsLoggingEnabled { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets value indicating whether distributed tracing spans are going to be created for this clients methods calls and HTTP calls.
+        /// </summary>
         public bool IsDistributedTracingEnabled { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets value indicating whether the "User-Agent" header containing <see cref="ApplicationId"/>, client library package name and version, <see cref="RuntimeInformation.FrameworkDescription"/>
+        /// and <see cref="RuntimeInformation.OSDescription"/>.
+        /// The default value can be controlled process wide by setting `AZURE_TELEMETRY_DISABLED` to true, false, 1 or 0.
```

Comment:
nit:  You may want to change out the back ticks for `<c></c>` blocks; those will render as fixed-width code for doc generation.   I don't believe that markdown gets encoded.